### ISSUE TITLE
Update colors for text hyperlink states

### ISSUE THREAD

**Issue by LibbyUX** [hubmapconsortium/hra-ui#1634]
# Summary
The color contrast of embedded links in text need to be improved across all UIs. `On Tertiary Fixed T-10` `B20A2F` does not work in relation to the colors chosen for body text and background fills in the design system.

# To do
- [x] Update design system implementation to use HRA `Tertiary T-40` `E00D3A` for the text hyperlink component as specified for each state
- [x] Update the design system text hyperlink arrow variant to only use `Tertiary T-40` `E00D3A` for both the text fill color and the arrow icon fill color

## Resources
- [Storybook: Text hyperlink](https://cdn.humanatlas.io/ui/storybook/design-system/?path=/docs/design-system-buttons-text-hyperlink--docs)
- [Figma: Design system repo](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/HRA-Design-System-Repository?node-id=8590-27601&t=1e8DHDOMZoCujBgU-4)

<img width="1673" height="778" alt="Image" src="https://github.com/user-attachments/assets/a27e98d6-70c2-47b0-a7a6-f133303b8b30" />
